_docs/bigquery/creating-bigquery-dataset.md

```diff
@@ -3,10 +3,26 @@ layout: page
 section: bigquery
 navigation_title: "Creating BigQuery dataset"
 title: "Creating BigQuery dataset"
-order: 4
-date: 2020-04-22
+order: 5
+date: 2020-04-29
 ---
 
 <!---
 In this article explained how to create Google BigQuery dataset
--->
\ No newline at end of file
+-->
+To store data in BigQuery you need to create a [dataset](https://cloud.google.com/bigquery/docs/datasets-intro){:target="_blank"}:
+
+1. Go to the [Google Cloud Platform Console](https://console.cloud.google.com/){:target="_blank"}.
+2. Open the console left side menu and click BigQuery.
+![](/img/bigquery_dataset.1.png)
+3. Select the project where you would like to create a dataset in the bottom-left menu and click CREATE DATASET.
+![](/img/bigquery_dataset.2.png)
+**Note: Instead of this screen, you might see general info about BigQuery:**
+![](/img/bigquery_dataset.3.png)
+In that case, you should [enable BigQuery API](enabling-bigquery-api).
+4. Enter Dataset ID (for example "SegmentStream"). All other options leave in default values.
```

Comment:
All labels should be bold. "SegmentStream" is a bad example, check original article